content/en/docs/concepts/configuration/assign-pod-node.md

```diff
@@ -92,6 +92,18 @@ For example, the value of `kubernetes.io/hostname` may be the same as the Node n
 and a different value in other environments.
 {{< /note >}}
 
+## Node isolation/restriction
+
+Adding labels to nodes allows targeting pods to specific nodes or groups of nodes.
+This could be used to ensure specific pods only run on nodes with certain security or regulatory properties.
+To ensure the labels are only set on the appropriate nodes, select labels that the kubelet processes running on the nodes cannot set on their own Node object.
+Otherwise, a compromised node could use its kubelet credentials, add labels to its `Node` object, and influence the scheduler to schedule pods to it incorrectly.
+
+To use labels in this way:
+1. Ensure you are using the <a href="/docs/reference/access-authn-authz/node/">Node authorizer</a> and have enabled the <a href="/docs/reference/access-authn-authz/admission-controllers/#noderestriction">NodeRestriction admission plugin</a>.
+2. Add labels under the `node-restriction.kubernetes.io/*` prefix to your nodes and use those labels in your node selectors.
```

Comment:
nit: since `node-restriction.kubernetes.io/*` is called a "prefix", there is no need to add the * at the end.